forum/qa-plugin/report-reason/frontend/src/ajaxService.js

```diff
@@ -0,0 +1,34 @@
+const AJAX_TIMEOUT_REASON = 'AJAX_TIMEOUT';
+const TIMEOUT = 5000;
+
+const URL = '/report-flag';
+const CONTENT_TYPE = 'application/json';
+
+function sendAjax(data) {
+	return new Promise((resolve, reject) => {
+		// TODO: handle timeout in better way - probably use AbortSignal API with some polyfill
```

Comment:
to zostaje? TODO na przyszłość czy zapomniane do poprawy przed wystawieniem?

Reply:
Zapisy TODO w kodzie zostawiłem specjalnie, aby pamiętać co i gdzie trzeba jeszcze poprawić przed zatwierdzeniem PRa. Część z nich opisałem w treści PRa.

Reply:
Ok, zwróciłem na wszystkie te komentarze uwagę dlatego, że jak napisałem: PR brzmi jak "chcę mergować te zmiany, sprawdźcie, to jest gotowe", a to wygląda trochę jak rzecz niegotowa. Wiadomo o co chodzi.